### ISSUE TITLE
“python_requires” should be set with “>=3.4”, as adafruit-circuitpython-busdevice 5.1.4 is not compatible with all Python versions.

### ISSUE THREAD

**Issue by PyVCEchecker** [adafruit/Adafruit_CircuitPython_BusDevice#81]
Currently, the keyword argument **python_requires** of **setup()** is not set, and thus it is assumed that this distribution is compatible with all Python versions. 
However, I found it is not compatible with **Python <3.4**. My local Python version is 2.7, and I encounter the following error when executing _“pip install adafruit-circuitpython-busdevice”_ 

```
Collecting adafruit-circuitpython-busdevice
  Downloading adafruit-circuitpython-busdevice-5.1.4.tar.gz (28 kB)
ERROR: Could not find a version that satisfies the requirement Adafruit-Blinka (from adafruit-circuitpython-busdevice) (from versions: none)
ERROR: No matching distribution found for Adafruit-Blinka (from adafruit-circuitpython-busdevice)
```
Dependencies of this distribution are listed as follows:
```bash
"Adafruit-Blinka"
```
I found that **Adafruit-Blinka** requires **Python>=3.4**, which results in installation failure of adafruit-circuitpython-busdevice in Python 2.7.

Way to fix:
modify **setup()** in **setup.py,** add **python_requires** keyword argument:
```python
setup(…
     python_requires=">=3.4"
     …)
```
Thanks for your attention.
Best regrads,
PyVCEchecker

**Comment by tekktrik** [comment]
Hi, this library actually went through some changes, and it actually requires Python 3.7+ now.  Thanks for finding this!

**Comment by dhalbert** [comment]
Fixed by #85.
